editor/utils/map.ts

```diff
@@ -67,3 +69,29 @@ export const useCurrentBounds = (
       [bounds[2], bounds[3]],
     ];
   }, [geojson]);
+
+/**
+ * Custom hook that manages basemap state and returns the appropriate basemap URL
+ * depending on the selected basemap type and app theme
+ */
+export const useBasemap = (initialBasemapType: "streets" | "aerial") => {
+  const [basemapType, setBasemapType] = useState<"streets" | "aerial">(
+    initialBasemapType
```

Comment:
thinking ahead, this `initialBasemapType` will allow us to pick what is the default basemap for various maps in the app, like in the crash details page it will be the Aerial map